client/src/modules/cohort/model/actions.js

```diff
@@ -1,9 +1,32 @@
+import { ENDPOINT_URL } from 'common/constants/variables';
 import { CohortActionTypes } from './actionTypes';
+import { getData, postData } from 'common/utils/fetchMethods';
 
 // Action creators
-const fetchCohortsSuccess = () => ({
-  type: CohortActionTypes.FETCH_COHORTS_SUCCESS
+const createCohortSuccess = data => ({
+  type: CohortActionTypes.CREATE_COHORT_SUCCESS,
+  payload: data
+});
+
+const fetchCohortsSuccess = data => ({
+  type: CohortActionTypes.FETCH_COHORTS_SUCCESS,
+  payload: data
 });
 
 // Dispatchers
-export const fetchCohorts = () => dispatch => dispatch(fetchCohortsSuccess());
+export const createCohort = (name, description, adminId) => dispatch =>
+  postData(`${ENDPOINT_URL}/cohorts/create`, {
```

Comment:
frondend does not need to know db structure - it should only send data and server will take care of use that data to create correct db entry.